### PR TITLE
Newsletters: add 60 (2019-08-21)

### PR DESCRIPTION
**Pull request by harding**
- [x] Finish compatibility matrix stuff
- [x] Check for C-Lightning release announcement

### CODE REVIEW

**Comment by moneyball** [comment]
initial review LGTM

File: _posts/en/newsletters/2019-08-21-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+- [Bitcoin Core #16383][] changes RPC commands that accept an
+  `include_watchonly` parameter so that it is automatically set to True
+  for wallets that have private keys disabled (i.e. that are only useful
+  as watch-only wallets).  This ensures results for watch-only addresses
+  are included in results.
+
+- [Bitcoin Core #15986][] extends the `getdescriptorinfo` RPC with an
+  additional `checksum` field that returns the checksum for the
+  descriptor provided by the user.  This differs from the checksum
```
Comment (jnewbery) [inline _posts/en/newsletters/2019-08-21-newsletter.md]: I don't think "This differs from the checksum described in Newsletter #34" is the clearest way to introduce this. Suggested alternative:

"Bitcoin Core #15986 extends the getdescriptorinfo RPC described in Newsletter #34. getdescriptorinfo returns the _normalized_ descriptor (which removes private keys and makes other modifications) along with its checksum. It now also returns a `checksum` field which gives the checksum of the _unnormalized_ descriptor provided by the user. Descriptor checksums follow a # character at the end of the string as described in the output script descriptor documentation."

File: _posts/en/newsletters/2019-08-21-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+## Bech32 sending support
+
+*Week 23 of 24 in a [series][bech32 series] about allowing the people
+you pay to access all of segwit's benefits.*
+
+The *News* section from this week's newsletter introduced a new feature
+on the Optech website.  {% include specials/bech32/23-compat.md %}
+
+## Notable code and documentation changes
```
Comment (jnewbery) [inline _posts/en/newsletters/2019-08-21-newsletter.md]: nit: add Bitcoin Core n16060 with one sentence referring to the news section.

**Comment by jnewbery** [comment]
ACK ef2778028e47385e3090375f9dc12bef9d27312f. Great stuff!

**Comment by practicalswift** [comment]
ACK ef2778028e47385e3090375f9dc12bef9d27312f

Thanks for yet another nice newsletter!

File: _posts/en/newsletters/2019-08-21-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  blocks where two previous soft forks activated have now been hardcoded
+  into Bitcoin Core as the point where those forks activate.  This
+  means any block chain reorganization that extends further back than
+  those blocks can create a chainsplit between nodes with this
+  hardcoding and those without it.  However, such a reorganization would
+  require an amount of proof of work roughly equal to the annual output
+  of all active Bitcoin miners (at the time of writing), so this is
+  considered to be both very unlikely and indicative of a threat that
+  could prevent consensus formation anyway.  The hardcoding, which is
+  similar to the [BIP90][] hardcoding made a couple years ago,
```
Comment (adamjonas) [inline _posts/en/newsletters/2019-08-21-newsletter.md]: a couple years -> a couple of years

**Comment by adamjonas** [comment]
ACK ef27780. Looks good!

**Comment by practicalswift** [comment]
ACK 44f6790c9cc55c67268d983e21b1adfc1de521af

**Comment by harding** [comment]
Rebased on master, addressed FIXMEs, and added an insight about bech32 support from the compat table now that it's all in one place (that 74% of the tools we surveyed support sending to a bech32 address).

File: _posts/en/newsletters/2019-08-21-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+  character at the end of the string as described in the [output script
+  descriptor][] documentation.
+
+- [Bitcoin Core #16060][] adds the hardcoded block heights for previous
+  soft forks as [described in the *news* section.][hc heights]
+
+- [LND #3391][] always returns the same error message for failed
+  payments in order to avoid leaking whether or not an invoice exists.
+  See [BOLTs #516][] for more details about the
+  information leak and the [BOLTs #608][] for a related leak.
```
Comment (jnewbery) [inline _posts/en/newsletters/2019-08-21-newsletter.md]:
```suggestion
  information leak and [BOLTs #608][] for a related leak.
```

**Comment by harding** [comment]
Added release announcement for C-Lightning 0.7.2.1.

**Comment by jnewbery** [comment]
ACK 6f9b93695ea557a6bc791daf8d9c2fc176bf03ea

**Comment by bitschmidty** [comment]
tACK 6f9b93695ea557a6bc791daf8d9c2fc176bf03ea

**Comment by jnewbery** [comment]
A newsletter merge without a rhyme?
@bitschmidty, surely you know that's a crime!

@moneyball, @harding and @adamjonas on the edge of their seats,
Waiting for your Wednesday verbal treats.

**Comment by bitschmidty** [comment]
@jnewbery Looks like you have taken the torch!

Optech announced the compatibility matrix
Some wallets gotta fix the basics

Soft fork blocks are now hardcoded
So consensus code doesnt get bloated

@harding great newsletter, @adamjonas and @jnewbery thanks for the reviews!

**Pull request merged**